webapp/store/views.py

```diff
@@ -126,19 +126,19 @@ def details_overview(entity_name):
 def details_docs(entity_name, slug=None):
     package = app.store_api.get_item_details(entity_name, fields=FIELDS)
     package = logic.add_store_front_data(package)
-    docs_url_prefix = f"/{package['name']}/docs"
 
-    # Fake package discourse topic
-    package["docs_topic"] = 3568
+    if not package["store_front"]["docs_topic"]:
+        abort(404)
```

Comment:
here instead of a 404 we should return a page that explains how to add docs 
Design is not done, so and empty page should be fine

https://zpl.io/2yrxd5y

Reply:
This is done!